apps/marketplace/components/BuyerDashboard/BuyerDashboard.scss

```diff
@@ -1,58 +1,207 @@
-.container {
-  margin-bottom: 100px;
-}
-
 .header {
   margin-bottom: 40px;
 }
 
+.organisation {
+  opacity: 0.6;
+}
+
+ul.menu {
+  box-sizing: content-box;
+  list-style-type: none;
+  margin-left: 0;
+  margin-top: 1.5rem;
+  padding-left: 0;
+  border-bottom: 1px solid #bebebe;
+}
+
+ul.menu li {
+  display: inline-block;
+  margin: 0;
+}
+
+ul.menu li a {
+  display: block;
+  font-size: 1.25rem;
+  padding: 10px;
+  margin-right: 7px;
+  border-bottom: 0;
+}
+
+ul.menu li a:hover {
+  background: transparent;
+}
+
+ul.menu li a.active {
+  font-weight: 700;
+  background-color: #def4f9;
+  border-bottom: solid 6px #17788d;
+}
+
 .headingRow {
-  font-size: 17px;
   font-weight: bold;
   color: #202751;
   border-bottom: solid 3px #202751;
   padding-bottom: 7px;
 }
 
-.priceRow {
-  padding-top: 20px;
-  padding-bottom: 21px;
-  border-bottom: solid 1px #777777;
+table.resultListing {
+  border-collapse: collapse;
+  margin-bottom: 40px;
+}
+
+table.resultListing tr {
+  border-bottom: solid 1px #bebebe;
   color: #313131;
 }
 
-.buttonsContainer {
-  float: right;
-  text-align: right;
-  margin-top: -4.5em;
+table.resultListing tr:nth-child(2n) {
+  background-color: #fafafa;
+}
+
+table.resultListing .colId {
+  width: 30px;
+  padding-left: 20px;
+}
+
+table.resultListing .colName {
+  width: 400px;
+}
+
+table.resultListingTeamOverview .colName {
+  padding-left: 20px;
+}
+
+table.resultListing .colClosing {
+  width: 180px;
+  white-space: nowrap;
+}
+
+table.resultListing .colAuthor {
+  width: 130px;
+}
+
+table.resultListing .colStatus {
+  width: 90px;
+}
+
+table.resultListing .colAction {
+  width: 300px;
+  padding-right: 20px;
+}
+
+table.resultListingTeamBriefs .colAction {
+  width: 180px;
+  padding-right: 20px;
+}
+
+table.resultListing .colAction a {
+  margin-right: 15px;
+  white-space: nowrap;
+}
+
+table.resultListing td, table.resultListing th {
+  text-align: left;
+  padding: 20px 40px 20px 0px;
+  box-sizing: content-box;
+}
+
+@media (max-width: 1195px) and (min-width: 900px) {
+  table.resultListing .colAction {
+    width: 140px;
+  }
+
+  table.resultListing td, table.resultListing th {
+    padding: 20px 30px 20px 0px;
+  }
+
+  table.resultListing .colAction a::after {
+    content: "\a";
+    white-space: pre;
+  }
+}
+
+@media (max-width: 899px) {
+  .header, table.resultListing {
+    margin-bottom: 20px;
+  }
+
+  table.resultListing thead, table.resultListing .empty, table.resultListing .colId, table.resultListing .colAuthor, .hideSmall {
+    display: none !important;
```

Comment:
As discussed, maybe check to see if `!important` is still required to override UI-Kit's rules .  If it is, a comment might be helpful here.